### ISSUE TITLE
Image not cache

### ISSUE THREAD

**Issue by lethianqt94** [swiftgif/SwiftGif#80]
Can my gif save in cache. I load a gif from url in a uiimageview in collectionviewcell. And when I scroll down, then up. I see it has to load again.

**Comment by yzhong52** [comment]
Yeah, this lib doesn't handle any image caching. You will have to handle the image caching part yourself I think.
